scripts/backup-restore.sh

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "$CK8S_CONFIG_PATH" ]; then
+  log "ERROR: CK8S_CONFIG_PATH is not set."
+  log "Please set and export the variable before running the script."
+  exit 1
+fi
```

Comment:
NIT: I think we should try to use same as we typically have

```suggestion
: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
```